### ISSUE TITLE
Queue?

### ISSUE THREAD

**Issue by dubcanada** [jackbrycesmith/laravel-caprover-template#6]
Hello,

Any ideas/suggestions on how to handle a queue worker? I assume the easiest would be to use supervisor and add another config setting similar to how the horizon is handled.

But I think a better solution would be to have a separate container for the queue worker?

Anyone figure that part out?

**Comment by jackbrycesmith** [comment]
> I assume the easiest would be to use supervisor

Yeah thats the way I would do it, though I've only ever worked with queues via horizon. Is there a reason why you're not using it?

> a better solution would be to have a separate container for the queue worker

At first glance I can't see any benefits to having it separately, seems to unnecessarily complicate things. Am I missing something?

**Comment by dubcanada** [comment]
I guess the only benefits of having it separate is running more than a single queue worker, or doing more advanced stuff such as queue specific memory/cpu restrictions or what not.

From a docker point of few, having the site and queue in the same container seems wrong from a separation point of view.
